labs/3-Add-Identity/README.md

```diff
@@ -89,25 +98,11 @@ You can read more about [selecting an identity management solution for ASP.NET C
 
 Now that our Keycloak instance is setup as an IdP, we can configure the web site to use it for identity and authentication purposes via OpenID Connect.
 
-1. Open the `WebApp` project and reference to the `Microsoft.AspNetCore.Authentication.OpenIdConnect` NuGet package, version `8.0.1`. You can use the `dotnet` CLI, Visual Studio NuGet Package Manager, or just edit the .csproj file manually:
+1. Open the `WebApp` project and add a reference to the `Microsoft.AspNetCore.Authentication.OpenIdConnect` NuGet package, version `8.0.1`. You can use the `dotnet` CLI, Visual Studio NuGet Package Manager, or just edit the .csproj file manually:
 
     ```xml
     <PackageReference Include="Microsoft.AspNetCore.Authentication.OpenIdConnect" Version="8.0.1" />
     ```
-1. In the `eShop.ServiceDefaults` project, create a new file called `ClaimsPrincipalExtensions.cs` and add the following extension methods class. These methods will make it easy to retrieve the user ID and name when needed from any of our projects:
-
-    ```csharp
-    namespace System.Security.Claims;
-
-    public static class ClaimsPrincipalExtensions
-    {
-        public static string? GetUserId(this ClaimsPrincipal principal)
-            => principal.FindFirst("sub")?.Value;
-
-        public static string? GetUserName(this ClaimsPrincipal principal) =>
-            principal.FindFirst(x => x.Type == "name")?.Value;
-    }
-    ```
```

Comment:
It was already there?

Reply:
Yeah - https://github.com/dotnet-presentations/eshop-app-workshop/blob/main/labs/3-Add-Identity/src/eShop.ServiceDefaults/ClaimsPrincipalExtensions.cs